src/StatusQ/Controls/StatusPickerButton.qml

```diff
@@ -31,82 +30,48 @@ Button {
 
     implicitWidth: 446
     implicitHeight: 44
-    background: Item {
-        anchors.fill: parent
-        Rectangle {
-            id: background
-            anchors.fill: parent
-            radius: 8
-            color: root.bgColor
-        }   
+    font.pixelSize: 15
+    horizontalPadding: 16
+    spacing: 4
+    background:Rectangle {
+        radius: 8
+        color: root.bgColor
     }
-
-    contentItem: Item {
-        anchors.fill: parent
-        state: root.type === StatusPickerButton.Type.Next ? "NEXT" : "DOWN"
-
-        RowLayout {
-            id: rowLabel
-            width: parent.width - icon.width - icon.anchors.rightMargin - icon.anchors.leftMargin
-            anchors.verticalCenter: parent.verticalCenter
+    contentItem: RowLayout {
+        clip: true
+        spacing: root.spacing
+        StatusIcon {
+            icon: "tiny/chevron-down"
+            visible: root.type === StatusPickerButton.Type.Down
+            color: !Qt.colorEqual(root.contentColor, Theme.palette.baseColor1) ? root.contentColor : Theme.palette.directColor1
+            width: 16
+            height: 16
```

Comment:
Just a suggestion: `Button` has an `icon.width` and `icon.height` properties; you could set the defaults at the toplevel and use them here